run.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/python3
 from ubuntuStation import app, db
-#from flask import Flask
+import os
 
+os.environ['OAUTHLIB_INSECURE_TRANSPORT'] = '1'
 app.run(debug=True, host='0.0.0.0', port=8989)
```

Comment:
Is this port here safe (i.e. it is not matching with the apache one, right?)? Also, what is this `insecure transport` equals true about? Looks fishy!